docs/reference/ml/anomaly-detection/apis/put-datafeed.asciidoc

```diff
@@ -73,6 +73,9 @@ those same roles.
   (Required, array) An array of index names. Wildcards are supported. For
   example: `["it_ops_metrics", "server*"]`.
 
+NOTE: If any indices are in remote clusters then `cluster.remote.connect` must
+not be set to `false` on any ML node.
```

Comment:
To be tied to the previous item I think this needs to be:

```
+
--
NOTE: If any indices are in remote clusters then `cluster.remote.connect` must
not be set to `false` on any ML node.
--
```

I.e. `+` and `--` before and `--` after.